lib/rutema/core/engine.rb

```diff
@@ -29,10 +45,16 @@ def initialize configuration
       @dispatcher=Dispatcher.new(@queue,configuration)
       @configuration=configuration
     end
-    #Parse, run, report
+
+    ##
+    # Parse the test specifications, execute the test(s) and report the results
+    #
+    # * +test_identifier+ - an optional identifier of a single test case to be
+    #   executed, this cannot be an arbitrary one but must still be contained
```

Comment:
I'm not really following the intention here for what you want to say about test_identifier. In particular about what you mean about not being arbitrary.

Reply:
Indeed I'm not so happy with my wording either. It's always difficult to summarize complex conditions in short sentences.

The point about the optional `test_identifier` is, that it must be specified somewhere in the configuration too. I cannot specify any `test_identifier` here, even if it were a valid one.